backend/api/Controllers/RobotController.cs

```diff
@@ -234,16 +231,10 @@ public async Task<ActionResult<RobotResponse>> DeleteRobot([FromRoute] string id
                 return NotFound(errorMessage);
             }
 
-            robot.Status = robotStatus;
-
             try
             {
-                var updatedRobot = await robotService.Update(robot);
-                var robotResponse = new RobotResponse(updatedRobot);
-
-                logger.LogInformation("Successful PUT of robot to database");
```

Comment:
And what is the reason for not logging it?

Reply:
Added logging back.